geps/gep-713.md

```diff
@@ -1345,6 +1334,13 @@ type RouteRule struct {
 }
 ```
 
+If a `sectionName` is specified, but does not exist on the targeted object, the Policy must fail to attach,
+and the policy implementation should record a `resolvedRefs` or similar Condition in the Policy's status.
+
+Note that the `sectionName` is currently intended to be used only for Direct Policy Attachment. Inherited Policies
+are always applied to the entire object. The `DirectPolicyTargetReference` API can be used to apply a direct Policy
+to a section of an object.
```

Comment:
This is great, but we will probably also need to update the parts about Direct Attachment further up in this document with the details that you should use the DirectTargetRef instead of the more generic one.

(Non-blocking, just a note for myself for the next time I touch this GEP).